enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -775,7 +775,25 @@ To answer this question, we need to do the following:
 * Verify that OpenShift Service Mesh will include support for this feature in time for this EP.
 * Evaluate any potential security concerns around this feature.
```

Comment:
Can you speak about the security concerns here?

Reply:
Good idea. Done.

Reply:
I'm not sure that CVE is the best representation of the issue, but we can leave it up to the reader's imagination.